### PR TITLE
[zh-cn]: sync the translation of `@document`

### PR DESCRIPTION
**Pull request by T34-active**
### Related issues and pull requests
commit: [592f6ec](https://github.com/mdn/content/commit/592f6ec42e54981b6573b58ec0343c9aa8cbbda8)
Relates to [#28158](https://github.com/mdn/content/pull/28158)

### CODE REVIEW

File: files/zh-cn/web/css/@document/index.md [hunk 1]
```diff
@@ -20,27 +22,55 @@ slug: Web/CSS/@document
...
+  - : 匹配文档的 URL 是否以参数指定的值开头。
+- `domain()`
+  - : 匹配文档的域名是否为参数中指定的域名或者为它的子域名。
+- `media-document()`
+  - : 根据参数中的字符串匹配媒体查询，匹配其中之一：`video`、 `image`、 `plugin` 或 `all`。
+- `regexp()` {{Deprecated_Inline}} {{Non-standard_Inline}}
+  - : 匹配文档的 URL 是否和参数中指定的 [正则表达式](/zh-CN/docs/Web/JavaScript/Guide/Regular_expressions) 匹配，该表达式必须匹配整个 URL。
 
-提供给 `url()`、`url-prefix()`，和 `domain()` 函数的参数可以不使用引号括起来。但提供给 `regexp()` 函数的参数必须用引号括起来。
+提供给 `url()`、 `url-prefix()`、 `domain()` 和 `media-document() ` 函数的参数可以不使用引号括起来。但提供给 `regexp()` 函数的参数必须用引号括起来。
```
Comment (skyclouds2001) [inline files/zh-cn/web/css/@document/index.md:36]:
```suggestion
提供给 `url()`、`url-prefix()`、`domain()` 和 `media-document()` 函数的参数可以不使用引号括起来。但提供给 `regexp()` 函数的参数必须用引号括起来。
```

File: files/zh-cn/web/css/@document/index.md [hunk 2]
```diff
@@ -1,11 +1,13 @@
...
 slug: Web/CSS/@document
+l10n:
+  sourceCommit: 592f6ec42e54981b6573b58ec0343c9aa8cbbda8
 ---
 
-{{CSSRef}} {{ SeeCompatTable}}
+{{CSSRef}}{{deprecated_header}}{{Non-standard_header}}
 
-`@document` [CSS](/zh-CN/docs/Web/CSS) [at-rule](/zh-CN/docs/Web/CSS/At-rule) 根据文档的 URL 限制其中包含的样式规则的作用范围。它主要是为用户定义的样式表（UserStyle）而设计的，但也可以在作者定义的样式表上使用。
+**`@document`** [CSS](/zh-CN/docs/Web/CSS) [At 规则](/zh-CN/docs/Web/CSS/At-rule) 根据文档的 URL 限制其中包含的样式规则的作用范围。它主要是为用户定义的样式表（User Style）而设计的，但也可以在作者定义的样式表上使用。
```
Comment (skyclouds2001) [inline files/zh-cn/web/css/@document/index.md:10]:
```suggestion
**`@document`** [CSS](/zh-CN/docs/Web/CSS) [At 规则](/zh-CN/docs/Web/CSS/At-rule)根据文档的 URL 限制其中包含的样式规则的作用范围。它主要是为用户定义的样式表（User Style）而设计的，但也可以在开发者定义的样式表上使用。
```

File: files/zh-cn/web/css/@document/index.md [hunk 3]
```diff
@@ -20,27 +22,55 @@ slug: Web/CSS/@document
...
+- `url()`
+  - : 匹配完全相符的 URL。
+- `url-prefix()`
+  - : 匹配文档的 URL 是否以参数指定的值开头。
+- `domain()`
+  - : 匹配文档的域名是否为参数中指定的域名或者为它的子域名。
+- `media-document()`
+  - : 根据参数中的字符串匹配媒体查询，匹配其中之一：`video`、 `image`、 `plugin` 或 `all`。
+- `regexp()` {{Deprecated_Inline}} {{Non-standard_Inline}}
+  - : 匹配文档的 URL 是否和参数中指定的 [正则表达式](/zh-CN/docs/Web/JavaScript/Guide/Regular_expressions) 匹配，该表达式必须匹配整个 URL。
```
Comment (skyclouds2001) [inline files/zh-cn/web/css/@document/index.md:34]:
```suggestion
- `domain()`
  - : 匹配文档的域名是否为参数中指定的域名或者为它的子域名。
- `media-document()`
  - : 根据参数中的字符串匹配媒体查询，匹配其中之一：`video`、`image`、`plugin` 或 `all`。
- `regexp()` {{Deprecated_Inline}} {{Non-standard_Inline}}
  - : 匹配文档的 URL 是否和参数中指定的[正则表达式](/zh-CN/docs/Web/JavaScript/Guide/Regular_expressions)匹配，该表达式必须匹配整个 URL。
```

File: files/zh-cn/web/css/@document/index.md [hunk 4]
```diff
@@ -20,27 +22,55 @@ slug: Web/CSS/@document
...
 
-提供给 `url()`、`url-prefix()`，和 `domain()` 函数的参数可以不使用引号括起来。但提供给 `regexp()` 函数的参数必须用引号括起来。
+提供给 `url()`、 `url-prefix()`、 `domain()` 和 `media-document() ` 函数的参数可以不使用引号括起来。但提供给 `regexp()` 函数的参数必须用引号括起来。
 
-提供给 `regexp()` 函数的正则表达式中的转义字符必须再次进行一次 CSS 转义。例如，一个点号（`.`），在正则表达式中匹配任意换行符之外的字母。如果想要匹配一个正真的点号，必须首先按照正则表达式的规则转义一次（变为 `\.`）然后在使用 CSS 的规则再转义一次（转换为`\\.`）。
+提供给 `regexp()` 函数的正则表达式中的转义字符必须再次进行一次 CSS 转义。例如，一个句号（`.`）在正则表达式中匹配任何字符。如果想要匹配一个真正的句号，必须首先按照正则表达式的规则转义一次（变为 `\.`）然后再使用 CSS 规则对该字符串进行转义（转换为 `\\.`）。
+
+`@document` 目前只在 Firefox 火狐浏览器中支持。如果你想在非 Firefox 火狐浏览器中使用此功能，你可以尝试使用由 @An-Error94 提供的这个 [this polyfill](https://github.com/An-Error94/Handy-Scripts/tree/master/%40document-polyfill)，它使用了用户脚本 [data-\*](/zh-CN/docs/Web/HTML/Global_attributes/data-*) 属性和 [属性选择器](/zh-CN/docs/Web/CSS/Attribute_selectors) 的组合。
```
Comment (skyclouds2001) [inline files/zh-cn/web/css/@document/index.md:40]:
```suggestion
`@document` 目前只在 Firefox 火狐浏览器中支持。如果你想在非 Firefox 火狐浏览器中使用此功能，你可以尝试使用由 @An-Error94 提供的这个 [polyfill](https://github.com/An-Error94/Handy-Scripts/tree/master/%40document-polyfill)，它使用了用户脚本 [data-\*](/zh-CN/docs/Web/HTML/Global_attributes/data-*) 属性和 [属性选择器](/zh-CN/docs/Web/CSS/Attribute_selectors) 的组合。
```

File: files/zh-cn/web/css/@document/index.md [hunk 5]
```diff
@@ -20,27 +22,55 @@ slug: Web/CSS/@document
...
+
+> **注意：** 这个属性还有一个带有 `-moz-` 的前缀版本 — `@-moz-document`。在 Firefox 火狐浏览器 59 Nightly 和 Beta 版本中，此版本已被限制仅在用户和 UA 样式表中使用 — 这是一个旨在减轻潜在 CSS 注入攻击的实验性功能。（请参阅 [Firefox bug 1035091](https://bugzil.la/1035091)）。
```
Comment (skyclouds2001) [inline files/zh-cn/web/css/@document/index.md:42]:
```suggestion
> **注意：** 这个属性还有一个带有 `-moz-` 的前缀版本——`@-moz-document`。在 Firefox 火狐浏览器 59 Nightly 和 Beta 版本中，此版本已被限制仅在用户和 UA 样式表中使用——这是一个旨在减轻潜在 CSS 注入攻击的实验性功能。（请参阅 [Firefox bug 1035091](https://bugzil.la/1035091)）。
```

File: files/zh-cn/web/css/@document/index.md [hunk 6]
```diff
@@ -52,13 +82,12 @@ slug: Web/CSS/@document
...
 
 ## 浏览器兼容性
 
 {{Compat}}
 
 ## 参见
 
-- [Per-site user style sheet rules](https://lists.w3.org/Archives/Public/www-style/2004Aug/0135) on the www-style mailing list.
-- The file [`userContent.css`](https://www.mozilla.org/support/firefox/edit#content) is a user stylesheet on Gecko-based browsers.
+- [Per-site user style sheet rules](https://lists.w3.org/Archives/Public/www-style/2004Aug/0135) 在 www-style 邮件列表上。
```
Comment (skyclouds2001) [inline files/zh-cn/web/css/@document/index.md:93]:
```suggestion
- [各网站用户样式表规则](https://lists.w3.org/Archives/Public/www-style/2004Aug/0135)在 www-style 邮件列表上。
```

**Review by yin1999 — APPROVED** [review]
Thank you @T34-active

**Comment by T34-active** [comment]
> Thank you @T34-active

感谢你提供了详细的翻译

**Pull request merged**